assets/js/src/customizer-preview/app.js

```diff
@@ -464,11 +464,17 @@ window.addEventListener('load', function () {
 									newValue.lineHeight &&
 									newValue.lineHeight[device]
 								) {
-									style += `line-height:${
-										newValue.lineHeight[device]
-									}${
-										newValue.lineHeight.suffix[device] || ''
-									};`;
+									let suffix = '';
+									if (
+										newValue.lineHeight.suffix[device] &&
+										newValue.lineHeight.suffix[device] !==
+											'em'
```

Comment:
Wouldn't this chained condition take effect on all line-height controls during live refresh? As far as I can tell we're only removing it for the body line-height for now. 😄 

I can see that the [general typography control](https://github.com/Codeinwp/neve/blob/3430a50f4ff32192ab356862d19c7668ee9a4aaf/inc/customizer/options/typography.php#L120-L143) was missed when converting everything to CSS variables on live refresh, as opposed to the [headings controls](https://github.com/Codeinwp/neve/blob/3430a50f4ff32192ab356862d19c7668ee9a4aaf/inc/customizer/options/typography.php#L242-L287). 

We should probably address that too, as it would reflect the changes on the frontend more accurately. 👍🏻